pkg/agent/multicast/mcast_controller.go

```diff
@@ -370,7 +380,11 @@ func (c *Controller) Run(stopCh <-chan struct{}) {
 		// Process multicast Group membership report or leave messages.
 		go wait.Until(c.worker, time.Second, stopCh)
 	}
-	go c.mRouteClient.run(stopCh)
+	if !c.flexibleIPAMEnabled {
+		// No need to route multicast traffic from or to the Node's transport interface because When Antrea FlexibleIPAM is enabled,
+		// that interface is connected to the OVS bridge.
+		go c.mRouteClient.run(stopCh)
```

Comment:
The goroutine doesn't run in case of flexibleIPAM, I don't think the problem is resolved.

Reply:
updated